chitterView.js

```diff
@@ -0,0 +1,35 @@
+const ChitterModel = require('./chitterModel')
+const ChitterApi = require('./chitterApi')
+
+class ChitterView {
+  constructor(model, api) {
+    this.model = model;
+    this.api = api;
+
+    this.mainContainerEl = document.querySelector('#main-container');
+    this.buttonEl = document.querySelector('#show-peeps-button');
+ 
+    // this.buttonEl.addEventListener('click', () => {
+      this.api.getChitterData(data => {
+        console.log(data)
+        console.log(this.model.getPeeps())
+      });
+    // }); button to only show all peeps when clicked
+  }
+
+  add() {
+    // adds a peep to the server
+  }
+
+  display() {
+    const peeps = this.model.getPeeps()
+
+    peeps.forEach(element => {
+      const peepEl = document.createElement('div')
+      peepEl.innerText = element.body
+      this.mainContainerEl.append(peepEl)
```

Comment:
If you set the peepEl to have a property `.className` just under inner text. You can then search for them more easily later with `querySelector()` - for example if you want to clear them off the screen.